src/drawer.canvasentry.js

```diff
@@ -157,6 +157,21 @@ export default class CanvasEntry {
         }
     }
 
+    /**
+     * Set the canvas transforms for wave and progress
+     *
+     * @param {boolean} vertical Whether to render vertically
+     */
+    applyCanvasTransforms(vertical) {
+        if (vertical) {
```

Comment:
add a comment for params passed to setTransform, it's black magic now ;)

Reply:
Oops! I had one in a previous iteration but must have lost it in the shuffle.